### ISSUE TITLE
Vega-cli does not work

### ISSUE THREAD

**Issue by domoritz** [vega/vega#1641]
```
yarn add vega-cli
echo '{"data": {"transform": [{"type": "formula"}]}}' | node_modules/.bin/vg2svg
```

```
Error: Unrecognized transform type: "formula"
    at Object.error (/private/tmp/node_modules/vega-util/build/vega-util.js:22:11)
    at parseTransform (/private/tmp/node_modules/vega-parser/build/vega-parser.js:1568:24)
    at /private/tmp/node_modules/vega-parser/build/vega-parser.js:2378:25
    at Array.forEach (<anonymous>)
    at parseData$1 (/private/tmp/node_modules/vega-parser/build/vega-parser.js:2377:22)
    at /private/tmp/node_modules/vega-parser/build/vega-parser.js:2915:7
    at Array.forEach (<anonymous>)
    at parseSpec (/private/tmp/node_modules/vega-parser/build/vega-parser.js:2914:31)
    at parseView (/private/tmp/node_modules/vega-parser/build/vega-parser.js:2989:5)
    at Object.parse (/private/tmp/node_modules/vega-parser/build/vega-parser.js:3658:12)
```

**Comment by domoritz** [comment]
The issue is that transforms is not populated.

**Comment by jheer** [comment]
Thanks. I'm aware of the issue, fix forthcoming...

**Comment by jheer** [comment]
Fixed with the release of vega-parser 5.1.1.
